mixer/adapter/solarwinds/internal/appoptics/batching.go

```diff
@@ -17,18 +17,30 @@ package appoptics
 import (
 	"time"
 
+	"sync/atomic"
```

Comment:
you typically don't want to use atomic directly, it's probably best to just use a mutex, it's rare you can solve anything with a single atomic (except low performance uuid generation)

https://golang.org/pkg/sync/atomic/

"These functions require great care to be used correctly. Except for special, low-level applications, synchronization is better done with channels or the facilities of the sync package."